### PR TITLE
https://issues.redhat.com/browse/ACM-6511 DR4Hub links update 2.9

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.8 and 2.7

### CODE REVIEW

File: business_continuity/backup_restore/backup_adv_config.adoc [hunk 1]
```diff
@@ -55,4 +55,9 @@ spec:
             memory: 256Mi
 ----
 
-Refer to the link:https://github.com/openshift/oadp-operator/blob/master/docs/config/resource_req_limits.md[Velero resource requests and limits customization] to find out more about the `DataProtectionApplication` parameters.
\ No newline at end of file
+[#additional-resources-dr4hub-adv]
+== Additional resources
+
+- Refer to the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/backup_and_restore/application-backup-and-restore#default-velero-cloud-provider-plugins[Default Velero cloud provider links] topic in the {ocp} documentation to learn more about the `DataProtectionApplication` parameters.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_adv_config.adoc]: My bad, I put the wrong link, and as a result you also changed the text description

This is the link https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/backup_and_restore/application-backup-and-restore#oadp-setting-resource-limits-and-requests_installing-oadp-aws

So the text should be the same as before, just change the link

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_adv_config.adoc (hunk 1 above)
Comment (oafischer) [reply business_continuity/backup_restore/backup_adv_config.adoc]: Updated, check if it's OK now. Thanks!

**Review by birsanv — CHANGES_REQUESTED** [review]
The link looks good now but you removed the second section :) requested in the task

[ ]- Add the following after the phrase  Refer to the [Velero resource requests and limits customization](https://github.com/openshift/oadp-operator/blob/master/docs/config/resource_req_limits.md) to find out more about the DataProtectionApplication parameters :

_Refer to the CPU and memory requirement for configurations([https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#cpu-and-memory-requirement-for-configurations|http://example.com/) for details on backup and restore CPU and memory requirements based on cluster usage.|http://example.com]%29/]_

**Comment by oafischer** [comment]
Whoops, should be fixed now @birsanv :)

File: business_continuity/backup_restore/backup_adv_config.adoc [hunk 2]
```diff
@@ -55,4 +55,9 @@ spec:
             memory: 256Mi
 ----
 
-Refer to the link:https://github.com/openshift/oadp-operator/blob/master/docs/config/resource_req_limits.md[Velero resource requests and limits customization] to find out more about the `DataProtectionApplication` parameters.
\ No newline at end of file
+[#additional-resources-dr4hub-adv]
+== Additional resources
+
+- Refer to the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/backup_and_restore/application-backup-and-restore#oadp-setting-resource-limits-and-requests_installing-oadp-aws[Setting Velero CPU and memory resource allocations] topic in the {ocp} documentation to find out more about the `DataProtectionApplication` parameters.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_adv_config.adoc]: This link is wrong

Should be  https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/backup_and_restore/application-backup-and-restore#default-velero-cloud-provider-plugins

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_adv_config.adoc (hunk 2 above)
Comment (oafischer) [reply business_continuity/backup_restore/backup_adv_config.adoc]: Third time's the charm :)

**Review by birsanv — APPROVED** [review]
lgtm thanks ! :)

**Review by dockerymick — APPROVED** [review]
links look good!

**Review by dockerymick — APPROVED** [review]

**Pull request merged**